### DISCUSSION TITLE
Closing Immortal PRs with renovate:embargo label

### DISCUSSION THREAD

**Discussion by siobh9** [renovatebot/renovate#29930] [Suggest an Idea]
### Tell us more.

Just bumping [this idea](https://github.com/renovatebot/renovate/issues/17985#issuecomment-2181060127) #17985 - would be super super helpful and esp avoid just a wild amount of excess builds clogging the pipeline from an Immortal PR rebasing on every push to the default branch.

**Comment by rarkins** [comment]
This needs a proper description/requirements in order to be able to be converted to an issue
